src/parallel/ocbbs.cpp

```diff
@@ -1079,7 +1079,8 @@ static Member_func members[] = {
 	"dt", thread_dt,
 	"t", nrn_thread_t,
 
-    "nrnbbcore_write", nrnbbcorewrite,
+    "nrnbbcore_write", nrncorewrite, // remove, deprecate, backward_compatible?
+    "nrncore_write", nrncorewrite,
```

Comment:
seems reasonable!

Reply:
So... leave as is or remove nrnbbcore_write altogether and no longer support second Vector arg?

Reply:
I guess not much harm to leave it in for backward compatibility. However we should remove `nrnbbcore_write` from docs and mention `nrncore_write` exclusively.

Reply:
Agree